README.md

```diff
@@ -1,4 +1,5 @@
 # Restor
+**Note**: Restor is deprecated, there is no intention to add macOS 10.15 support, or update the project going forward. 
```

Comment:
Not sure about the wording here - Restor runs on 10.15 and as far as I've seen can restore 10.15 images, is that incorrect?

Reply:
Yes, I should phrase it differently.